tests/integration/targets/ntnx_karbon_clusters_and_info/tasks/crud.yml

```diff
@@ -76,10 +76,11 @@
       name: "{{network.dhcp.name}}"
     cluster_type: DEV
     cni:
-      node_cidr_mask_size:  "{{cni.node_cidr_mask_size}}"
+      node_cidr_mask_size: "{{cni.node_cidr_mask_size}}"
       service_ipv4_cidr: "{{cni.service_ipv4_cidr}}"
       pod_ipv4_cidr: "{{cni.pod_ipv4_cidr}}"
       network_provider: Flannel
+    disable_monitoring: true
```

Comment:
Add a check in asserts to verify if this works or not. Also check in asserts of other tasks if disable_monitoring is by default set to false or not